hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/hdds/scm/TestCloseContainer.java

```diff
@@ -109,8 +106,7 @@ public void testReplicasAreReportedForClosedContainerAfterRestart()
       throws Exception {
     // Create some keys to write data into the open containers
     for (int i = 0; i < 10; i++) {
-      TestDataUtil.createKey(bucket, "key" + i, ReplicationConfig
-          .fromTypeAndFactor(ReplicationType.RATIS, ReplicationFactor.THREE),
+      TestDataUtil.createKey(bucket, "key" + i, null,
```

Comment:
No need to pass `null` as `ReplicationConfig`, since we have `createKey(OzoneBucket, String, byte[])`.

```suggestion
      TestDataUtil.createKey(bucket, "key" + i,
```

(Applies to all invocations where third parameter is `null`.)